### DISCUSSION TITLE
v3不能多轮对话

### DISCUSSION THREAD

**Discussion by Gary2018X** [acheong08/ChatGPT#1164] [Q&A] [answered]
v3不能多轮对话吗？

**Comment by acheong08** [comment, marked as answer]
能

**Reply by Gary2018X** [reply]
需要什么特殊设置吗？我直接ask好像不行

**Reply by acheong08** [reply]
什么问题？

**Reply by Gary2018X** [reply]
![image](https://user-images.githubusercontent.com/56428019/225288411-1d415acd-1629-4924-af05-17d08c4f0dcd.png)

**Reply by Gary2018X** [reply]
这显然是没有上下文的？

**Reply by acheong08** [reply]
![image](https://user-images.githubusercontent.com/36258159/225289142-870aaeb5-47b3-47f0-b6a7-b50fd6559029.png)

**Reply by Gary2018X** [reply]
确实奇怪 我再试试 谢谢

**Reply by Gary2018X** [reply]
这和我的版本有关系吗？我现在的版本是3.3.4

**Reply by acheong08** [reply]
👀 https://github.com/acheong08/ChatGPT/releases/tag/4.0.2

**Reply by Gary2018X** [reply]
是我犯了一个低级错误，我初始化chatbot放在了循环里面

**Comment by Gary2018X** [comment]
done
